### PR TITLE
changed height to rem from vh

### PR DESCRIPTION
**Pull request by msivasubramaniaan**
Signed-off-by: msivasubramaniaan [msivasub@redhat.com](mailto:msivasub@redhat.com)

Fix: https://github.com/redhat-developer/vscode-openshift-tools/issues/2930

**Post Fix:**
image

### CODE REVIEW

File: src/webview/welcome/app/welcomePage.tsx [hunk 1]
```diff
@@ -372,10 +372,10 @@ export class Welcome extends React.Component<DefaultProps, {
                         </div>
                         <section className='sticky-section-content'>
                             <div className='sticky-section-exten'>
-                                <p className='section__header-hint sticky-section-hint'>Allows developers to easily create, deploy and live debug container applications running on OpenShift &#38; Kubernetes. Thus enhancing the development inner loop workflow through One-click actions right from IDE.</p>
+                                <p className='section__header-hint sticky-section-hint'>Allows developers to easily <p className='highlight'>create, deploy and live debug</p> container applications running on OpenShift &#38; Kubernetes. Thus enhancing the development inner loop workflow through One-click actions right from IDE.</p>
```
Comment (datho7561) [inline src/webview/welcome/app/welcomePage.tsx:375]: This 'highlight' class doesn't seem to change the text that's in the the `<p>` tag

**Comment by datho7561** [comment]
It looks like the div containing the "OpenShift extension" title on the left is a fixed height (based on `vh`), so it scrolls past the paragraphs. Is there a way to set it to scale its height to the height of the paragraphs?

LeftTitleBoxIsFixedLength

**Comment by msivasubramaniaan** [comment]
> It looks like the div containing the "OpenShift extension" title on the left is a fixed height (based on `vh`), so it scrolls past the paragraphs. Is there a way to set it to scale its height to the height of the paragraphs?
> 
> LeftTitleBoxIsFixedLength

May I know your screen-size and the resolution? I couldn't able to reproduce on my system

**Review by datho7561 — APPROVED** [review]
Looks good. I think there is a simpler way to accomplish this, but if you want to release soon, I think this can work for the time being

**Pull request merged**